sdk/__init__.py

```diff
@@ -48,23 +55,30 @@ def create_batch(
         Args:
             serialized_sequence: Serialized pulser sequence.
             jobs: List of jobs to be added to the batch at creation. (#TODO: Make optional after Iroise MVP)
-            emulator: Whether to run the batch on an emulator.
-              If set to false, the device_type will be set to the one
+            device_type: The type of device to use, either an emulator or a QPU
+              If set to QPU, the device_type will be set to the one
               stored in the serialized sequence
             wait: Whether to wait for results to be sent back
 
 
         Returns:
             Batch: The new batch that has been created in the database.
         """
-        batch_rsp, jobs_rsp = self._client._send_batch(
-            {
-                "sequence_builder": serialized_sequence,
-                "emulator": emulator,
-                "webhook": self.webhook,
-                "jobs": jobs,
-            }
-        )
+
+        is_emulator = device_type != DeviceType.QPU
+
+        req = {
+            "sequence_builder": serialized_sequence,
+            "webhook": self.webhook,
+            "jobs": jobs,
+        }
+
+        # the emulator field is only added in the case
+        # an emulator job is requested otherwise it's left empty
+        if is_emulator:
```

Comment:
nit since `is_emulator` is only used here, either I would directly put : `if device_type != DeviceType.QPU` here or at least I would move the definition right above.

Reply:
Good point :)